phpunit.xml.dist

```diff
@@ -18,4 +18,7 @@
         </testsuite>
     </testsuites>
 
+    <listeners>
+        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>
```

Comment:
IIRC, this is not required is you launch the test suite with `vendor/bin/simple-phpunit`.

Reply:
big :-1: for using custom entry point, `simple-phpunit` wrapper does a lot of more things